ThreadGroup/Controllers/TGMainViewController.m

```diff
@@ -149,6 +149,7 @@ - (void)reloadTableView {
 #pragma mark - Main spinner
 
 - (void)hideMainSpinner {
+    [self.routerSearchView setUserInteractionEnabled:YES];
```

Comment:
You may want to setUserInteraction:NO in viewDidLoad instead of in the xib. Because its not clear here why you need to setUserInteraction:YES. Alternatively, you can put a comment on the code